scripts/load_tester/drill_config.yml

```diff
@@ -0,0 +1,24 @@
+concurrency: 4
+base: 'http://localhost:8000'
+iterations: 5
```

Comment:
I would do iteration 1 (as we do not want to do load testing, just fill the cache)